enterprise/internal/codemonitors/background/workers.go

```diff
@@ -99,11 +99,11 @@ func newActionJobResetter(_ context.Context, s edb.CodeMonitorStore, metrics cod
 			RecordResets:        metrics.resets,
 		},
 	}
-	return dbworker.NewResetter(workerStore, options)
+	return dbworker.NewResetter(logger.Scoped("dbworker.Resetter", ""), workerStore, options)
```

Comment:
```suggestion
	return dbworker.NewResetter(logger, workerStore, options)
```
inline with the others,  the scope gets defined earlier and a scoped logger is passed here